README.md

```diff
@@ -1374,163 +2341,257 @@ iam_user { 'bob':
 ```
 
 #### Type: kms
-The `kms` type manages KMS key lifecycle and their policies.  The name of the
-resource is prefixed with `alias/` to set the alias of the KMS key, since keys
-themselves don't have any notion of name, outside of an attached alias.
 
-``` Puppet
+The 'kms' type manages KMS key lifecycle and their policies.  The name of the resource is prefixed with 'alias/' to set the alias of the KMS key, since keys themselves don't have any notion of name, outside of an attached alias.
+
+```puppet
 kms { 'somekey':
   ensure => present,
   policy => template('my/policy.json'),
 }
 ```
 
-The above resource may be viewable elsewhere as `alias/somekey`.
+The above resource may be viewable elsewhere as 'alias/somekey'.
 
 #####`policy`
+
 The JSON policy document to manage on the given KMS key.
 
 #### Type: rds_db_parameter_group
 
-Note that currently, this type can only be listed via puppet resource,
-but cannot be created by Puppet.
+Note that currently, this type can only be listed via puppet resource, but cannot be created by Puppet.
```

Comment:
`puppet resource` is a command, so should be in backticks. (Not that you should have seen it!)